### PR TITLE
Add Bambu Lab icon

### LINKED ISSUE #10985: Request: Bambu Lab
### Brand Name

Bambu Lab

### Website

https://bambulab.com

### Popularity Metric

The Similarweb rank is 7,442. See https://www.similarweb.com/website/bambulab.com

### Official Resources for Icon and Color

svg: https://www.printables.com/model/367329-bambu-lab-logos-svg/files - "Bambu Lab Logo Only_SVG.svg"
color:  00ae42

### Additional Comments

Did not find official resources for logo but this one looks clean and can be freely used as per the CC license

### PR DESCRIPTION
**Pull request by RaghavMaliwal**
image

**Issue:** closes #10985.

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by uncenter** [comment]
Please use the provided pull request template in the future!

**Review by uncenter — CHANGES_REQUESTED** [review]
You need to follow our contributing guidelines when adding an icon: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon. The `source` you have provided is not official, and the icon itself is not optimized at all.

**Comment by RaghavMaliwal** [comment]
Hello.. @uncenter I have changed the svg file and i think it should work fine now...And the source of the svg I have attached is the one provided in the issues section that's why I pasted it as it is....now where should I find its official source...?

**Comment by uncenter** [comment]
> Hello.. @uncenter I have changed the svg file and i think it should work fine now

See the linter for why your current SVG is invalid. You also again need to read through the full contributing guidelines as there are quite a few steps you have missed.

> And the source of the svg I have attached is the one provided in the issues section that's why I pasted it as it is....now where should I find its official source...?

https://github.com/simple-icons/simple-icons/issues/10985#issuecomment-2124014386

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:1880]:
```suggestion
        {
            "title": "Bambu Lab",
            "hex": "00ae42",
            "source": "https://github.com/bambulab/BambuStudio/blob/master/resources/images/BambuStudio.svg" 
        },
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:1877]: There is also an SVG available on the homepage of the website. I'd suggest we use that as a source.
```suggestion
            "source": "https://bambulab.com" 
```

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:1876]:
```suggestion
            "hex": "00AE42",
```

File: icons/bambulab.svg
Comment (adamrusted) [inline icons/bambulab.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 24 24"><title>Bambu Lab</title><path d="M12.662 24V8.959l8.535 3.369V24zm-9.859-.003v-7.521l8.534-3.371-.001 10.892zM2.803 0h8.533l.001 11.672-8.534 3.369zm9.859 0h8.535v10.892l-8.535-3.371z"/></svg>
```

**Review by adamrusted — COMMENTED** [review]
Looks like the icon isn't centered - and the hex is lowercase.
I'll merge these changes, if another maintainer can review?

**Review by uncenter — APPROVED** [review]
Icon is a perfect match!

**Comment by RaghavMaliwal** [comment]
Hi @adamrusted @uncenter ,

Thank you for merging the PR and for correcting the flaws I had missed. I appreciate your guidance and the improvements made.

Additionally, I have another PR (#11013) in which I have received permission from Maze(https://github.com/simple-icons/simple-icons/pull/11013#issuecomment-2144621025. I would be grateful if you could review it at your earliest convenience.

Thanks again for your support!

Best regards,
@RaghavMaliwal

**Pull request merged**
